### ISSUE TITLE
The KI and pronouns

### ISSUE THREAD

**Issue by DoobesURU** [H-uru/Plasma#1135]
A rather important question came up today at the dev/story talk, and that was the issue of pronouns and ability to choose/set/reset them.

With that in mind, could there be a way to select or even customize one's emote pronouns within the KI?

**Comment by dpogue** [comment]
Right now the pronouns are presumably based on the (binary) gender of the avatar. It's unlikely (but not technically impossible) that someone could add a 3rd gender-neutral avatar, so let's put that option aside for the moment.

It should be feasible to provide a user-selected pronoun, but we are probably limited in how free-form we could make it. I'm imagining something like this:

```
0 = kDefault (matches the avatar gender, default for all existing players)
1 = kMale (he/him/his)
2 = kFemale (she/her/her)
3 = kNeutral (they/them/their)
```

We could probably store this in the PlayerInfoNode in a UInt32 field that's not currently being used. Right now it looks like the "avatar shape" (aka gender) is stored as a string on the PlayerNode.

While you mention customizing in the KI, I think the pronoun selection should actually be part of the avatar customization GUI.

**Comment by ZarothYe** [comment]
I am good with this idea, seems nice. What would this affect visibly in-game other than the pronoun that is used in emote strings, e.g., "Zaroth shakes her head"? Are there any other things I'm not thinking of?

**Comment by Hoikas** [comment]
I would prefer for the game to be grammatically correct, so hopefully we can find a singular pronoun for `kNeutral`, eg ze.

**Comment by ZarothYe** [comment]
Singular "they" or "their" is acceptable, IMO, and is a relatively common choice of pronoun for non-binary individuals. It's been used as a singular pronoun for a long time when gender of the individual is unknown or unimportant (see definition 2):

![image](https://user-images.githubusercontent.com/94947971/165002212-ea38bdb2-12be-44e0-b657-9aa3a7bf4202.png)

**Comment by DamnBriggsy** [comment]
I understand this topic is hotly debated for multiple reasons, but singular they is considered grammatically correct. Most research into the subject concludes that the opposition to it is recent, relatively speaking. I am adding more sources to this for your (plural) consideration. I consider this to be a simple, obvious change.

[https://apastyle.apa.org/style-grammar-guidelines/grammar/singular-they](https://apastyle.apa.org/style-grammar-guidelines/grammar/singular-they)
[https://www.merriam-webster.com/words-at-play/singular-nonbinary-they](https://www.merriam-webster.com/words-at-play/singular-nonbinary-they)
[https://www.thesaurus.com/e/grammar/they-is-a-singular-pronoun/](https://www.thesaurus.com/e/grammar/they-is-a-singular-pronoun/)
[https://blogs.illinois.edu/view/25/677177](https://blogs.illinois.edu/view/25/677177)

**Comment by dpogue** [comment]
Singular "they" predates the use of singular "you" in the English language, and is commonly accepted as a neutral third-person singular pronoun.

**Comment by keiyakins** [comment]
Are there any instances of the game using language that needs to change to agree? Stuff like "they is" is always jarring.

**Comment by dpogue** [comment]
> Are there any instances of the game using language that needs to change to agree? Stuff like "they is" is always jarring.

I think the only time pronouns are used in generated text in Uru is as the possessive form: "his hands", "her head", "their feet", etc.

I think swapping in "their" shouldn't cause any issues in those spots.

**Comment by ZarothYe** [comment]
> > Are there any instances of the game using language that needs to change to agree? Stuff like "they is" is always jarring.
> 
> I think the only time pronouns are used in generated text in Uru is as the possessive form: "his hands", "her head", "their feet", etc.
> 
> I think swapping in "their" shouldn't cause any issues in those spots.

This is correct. There is no nominative pronoun use in the emote text at this time. In addition, possessive use like this is only present in English, e.g., "Zaroth claps her hands" vs German's "Zaroth klatscht". The non-English LOCs do not even have translations for `<element name="Her">` and `<element name="His">`.
